vcpkg.json

```
@@ -1,7 +1,11 @@
 {
-  "builtin-baseline": "000d1bda1ffa95a73e0b40334fa4103d6f4d3d48",
+  "builtin-baseline": "f7423ee180c4b7f40d43402c2feb3859161ef625",
   "dependencies": [
     "gtest",
-    "zlib"
+    "zlib",
+    {
+      "name": "sdl2",
+      "version>=": "2.30.3"
+    }
   ]
```

an `override` section is needed to specify an _exact_ version, otherwise the `builtin-baseline` and `version>=` (note `>=`) fields together just designate a _minimum_ version
```suggestion
  ],
  "overrides": [
    {
      "name": "sdl2",
      "version": "2.30.1"
    }
  ]
```

---

https://learn.microsoft.com/en-us/vcpkg/reference/vcpkg-json#overrides

---

thanks for the tip, I added the override